docs/notebooks/demo_Cometary_Activity.ipynb

```diff
@@ -0,0 +1,1047 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c93503c5",
+   "metadata": {},
+   "source": [
+    "# Incorporating Cometary Activity"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c6a7190d",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to demonstrate the apply cometary activity within `Sorcha`.\n",
+    "\n",
+    "We will use the community tools part of the `Sorcha-addons`(https://github.com/dirac-institute/sorcha-addons) package\n",
+    "\n",
+    "The idea is that the user can, in principle, implement their own method for cometary activity, and incorporate them in their simulation. The goal of `Sorcha-addons` is for both the development team, as well as for the community, to share their implementations of custom coemtary activity models. "
```

Comment:
"and incorporate [it] in their simulation."

"custom [cometary] activity models."

Reply:
thanks!